app/models/cart.rb

```diff
@@ -57,7 +57,7 @@ def discounts(item)
   end
 
   def discount_applies
-    @contents.any? do |item_id|
+    @contents.any? do |item_id, quantity|
```

Comment:
Oops. Tricky mistake, was returning an array of the id and quantity which obvs couldn't be passed into an AR query